### ISSUE TITLE
[13일차] DoReFa-Net: Training Low Bitwidth Convolutional Neural Networks with Low Bitwidth Gradients

### ISSUE THREAD

**Issue by dhkim0225** [dhkim0225/1day_1paper#13]
https://arxiv.org/abs/1606.06160

20/04/17

**Comment by DK-sr** [comment]
이거 이제 안하나요?

**Comment by dhkim0225** [comment]
너무 바빠요 ㅠㅠ

**Comment by DK-sr** [comment]
이거 이제 안하나요?

**Comment by dhkim0225** [comment]
언제쯤 다시 할 수 있을까요... 우선은 cvpr 에 나온 OCR 논문부터 보고 레포 관리해야해요 ㅠㅠ

**Comment by DK-sr** [comment]
그것도 안하면서..
